h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -31,7 +31,7 @@
 
 /** Persistence backend for S3 */
 public final class PersistS3 extends Persist {
-  private static final String HELP = "You can specify a credentials properties file with the -aws_credentials command line switch.";
+  private static final Logger LOGGER = Logger.getLogger(PersistS3.class);
```

Comment:
I would rename this to LOG, same I used in xgboost

Reply:
Done.